src/dakota.py

```diff
@@ -43,42 +43,22 @@
     os.environ['LC_ALL'] = 'C'
 
 import pyDAKOTA
-import pyDAKOTA
-from mpi4py import MPI
-comm = MPI.COMM_WORLD
-
-# Hard-coded assumption regarding availability of MPI.
-if sys.platform in ('cygwin', 'win32'):
-    _HAVE_MPI = False
-else:
-    _HAVE_MPI = True
-
-# Dictionary to map from str(id(data)) to data object.
-_IDS = [-1] * (comm.Get_size() + 1) * 2
-_USER_DATA = [weakref.WeakValueDictionary()] * (comm.Get_size() + 1) * 2
-
 
 class DakotaBase(object):
     """ Base class for a DAKOTA 'driver'. """
 
     def __init__(self):
         self.input = DakotaInput()
 
-    def run_dakota(self, infile='dakota.in', mpi_comm=None, use_mpi=True,
+    def run_dakota(self, infile='dakota.in',
                    stdout=None, stderr=None, restart=0, other_model=None):
         """
         Write `self.input` to `infile`, providing `self` as `data`.
         Then run DAKOTA with that input, MPI specification, and files.
         DAKOTA will then call our :meth:`dakota_callback` during the run.
         """
-        if mpi_comm: comm = mpi_comm
-        else: comm = None
-        #if comm.Get_rank() == 0:
-        #self.input.write_input(infile, data=self)
-        if comm:
-            if comm.Get_rank() == 0: self.input.write_input(infile, data=self, other_data=None)
-        else: self.input.write_input(infile, data=self, other_data=None)
-        run_dakota(infile, mpi_comm, use_mpi, stdout, stderr, restart=restart)
+	self.input.write_input(infile, data=self, other_data=None)
```

Comment:
Sure, I missed this one.... I'll get it fixed and add a new commit. This file will get more attention in the next commits when going into testing and linking this with libseba.